csv-validation-tests/src/main/shell/GenerateTests.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env ruby
```

Comment:
Errr, I wouldn't call this a bash file so wouldn't use the `.sh` extension. It's ruby so I'd be sure to use `.rb`. You can still execute it in the same way.

Reply:
done